config/routes.rb

```diff
@@ -27,6 +32,11 @@ def collection_slug_constraint
     end
   end
 
+  # https://mlit.atlassian.net/browse/ARC-84
+  # ARC-84: Remove ban on periods in eadid field.
+  # Redirects for normalized URLs in the wild.
+  get '/catalog/:id', to: redirect { |path_params, req| "/catalog/#{ARC_84_EADID_MAP[path_params[:id]]}" }, constraints: lambda { |request| ARC_84_EADID_MAP.has_key?(request.path_parameters[:id]) }
```

Comment:
I get what you're doing, but where is `ARC_84_EADID_MAP` defined?

Reply:
Under `./config/initializers` https://github.com/gkostin1966/umich-arclight/blob/ARC-84/allow_periods_in_eadid_field/config/initializers/arc_84.rb

Reply:
Ugh, sorry --- of course that was hidden in the github UI and I missed a step pulling your branch down locally.

Reply:
Actually I force pushed it this morning because I removed the ARC_84_EADID_LIST since it was redundant.